xbmc/pvr/PVRManager.cpp

```diff
@@ -962,6 +962,7 @@ bool CPVRManager::CanSystemPowerdown(bool bAskUser /*= true*/) const
                                        CVariant{text},
                                        CVariant{222}, // "Shutdown anyway",
                                        CVariant{19696}, // "Cancel"
+                                       CVariant{ "" }, // No 3rd button
```

Comment:
why not use a convenience method without the custom button label parameter. In most cases we don't need a custom button.

Reply:
Because the parameters are variants or int I can not see how to differentiate between
`ShowAndGetInput(522, 20340, 106, 107, 1000)` as giving 
`heading, text, noLabel, yesLabel, autoCloseTime` or
`heading, text, noLabel, yesLabel, customLabel`
Yes we generally use `CVariant{19696}` as a parameter, but `19696` works too.

There are only a few calls that use auto close, and so not too inconvenient to have to give a blank custom button

Reply:
Or did you mean a new function in DialogHelper.cpp rather than an overlay?

Reply:
I counted 3 or 4 times, that you need autoclose and have to explicitly call it. So I did not mention it.

Reply:
So would you prefer me to leave the parameters to `ShowYesNoDialogText`  and `ShowYesNoDialogLines` unchanged, and add new convenience methods `ShowYesNoOtherDialogText` and `ShowYesNoOtherDialogLines` ?

Reply:
@Paxxi thinks it makes sense to have a new convenience method too, so going with that